app/routes/events/components/EventDetail/index.tsx

```diff
@@ -86,6 +88,26 @@ const InterestedButton = ({ isInterested }: InterestedButtonProps) => {
   );
 };
 
+const groupLogoDict: { [key: string]: string } = {
```

Comment:
Preferably just use the group logo, especially since other groups can also create events as well

Reply:
The get logo API is not working at this moment. I'll make an issue and maybe fix it myself.